### PR TITLE
feat(cicd): Extend default `cloudbuild.yaml` for App Engine deployments

### PR DESCRIPTION
**Pull request by Grashalmbeisser**
Add cloud logging options

### CODE REVIEW

**Review by phorward — APPROVED** [review]

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+# allow AppEngine Deployment
+# https://console.cloud.google.com/cloud-build/settings
+#
+# Make sure the projects cloudbuild service account has permission to download from
+#
+# Modify project Submodules
+# for public Repos:
+#     use https urls
+#     git config -f .gitmodules submodule.sources/less/ignite.url https://github.com/viur-framework/ignite.git
```
Comment (sveneberth) [inline cloudbuild.yaml:19]:
```suggestion
#     git config -f .gitmodules submodule.sources/less/ignite.url https://github.com/viur-framework/viur-ignite.git
```

File: cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+steps:
+  # clone source code
+  # if you need to reconfigure submodules to use https or git, do this here
+  - name: 'gcr.io/cloud-builders/git'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        git config -f .gitmodules submodule.sources/less/ignite.url https://github.com/viur-framework/ignite.git
```
Comment (sveneberth) [inline cloudbuild.yaml:30]:
```suggestion
        git config -f .gitmodules submodule.sources/less/ignite.url https://github.com/viur-framework/viur-ignite.git
```

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# https://source.cloud.google.com/
+#
+# create a trigger for main repository
+# https://console.cloud.google.com/cloud-build/triggers/add
+# add name (BuildDeploy) and description (Build & Deploy)
+
+# activate App Engine Admin api
+# https://console.developers.google.com/apis/api/appengine.googleapis.com/overview
+#
+# allow AppEngine Deployment
```
Comment (sveneberth) [inline cloudbuild.yaml:11]:
```suggestion
# activate App Engine Admin API
# https://console.developers.google.com/apis/api/appengine.googleapis.com/overview
#
# allow App Engine Deployment
```

**Review by sveneberth — COMMENTED** [review]
Hey, The ignite repo has been renamed for a while and the spelling of the google products wasnt correct. Beside this please make the branch mergeable, it has currently conflicts.

**Review by sveneberth — CHANGES_REQUESTED** [review]
@Grashalmbeisser  Please resolve the merge conflict

**Review by sveneberth — APPROVED** [review]

**Pull request merged**